### ISSUE TITLE
Hang when changing second folder password strength and required password strength

### ISSUE THREAD

**Issue by antoweb** [nilsteampassnet/TeamPass#2198]
### Steps to reproduce
1. Enter as admin and go to Manage Folder top menu button
2. Edit a folder and change the password stregth required from medium to Very Weak and chooose yes to both allow item creation ....
3. Save The folder settings, all OK
4. Repeat the procedure with another folder, change settings described above and click Save
5. The application hang with the gear rolling

### Expected behaviour
Same result for the first folder editing

### Actual behaviour
When edit the password strength for the second folder application hang and is needed to refresh the page before changing another folder

### Server configuration
**Operating system**: Windows 2008 R2

**Web server**: Apache

**Database:** Mysql

**PHP version:** 5.6

**Teampass version:**  2.27.11

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
PLEASE attach to this issue the file `/includes/config/tp.config.php`.

### Client configuration
**Browser:** All browsers

**Operating system:** All

#### Web server error log 
NO server php/apache error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by nilsteampassnet** [comment]
I confirm this ... I will fix in 2.1.27.12

THanks

**Comment by nilsteampassnet** [comment]
Fix will be committed today in development branch
Reopen on need
